### PR TITLE
i18n(hi): update `index.mdx`

### PR DESCRIPTION
**Pull request by at-the-vr**
#### Description

- add testimonials translation in https://starlight.astro.build/hi/

### CODE REVIEW

File: docs/src/content/docs/hi/index.mdx [hunk 1]
```diff
@@ -44,6 +46,107 @@ import AboutAstro from '~/components/about-astro.astro';
 	</Card>
 </CardGrid>
 
+<TestimonialGrid title="What people are saying">
```
Comment (HiDeoo) [inline docs/src/content/docs/hi/index.mdx:49]: The section title can be safely translated.

**Review by HiDeoo — COMMENTED** [review]
Thanks for the translation :raised_hands:

As a non-native speaker, I reviewed this PR using the [Docs Translation Review Checklist](https://hackmd.io/1KYnrpUQR0SAGTzHIUoUDg?view) and this looks great to me, I just left a tiny comment for the section title that can also be translated. :thumbsup:

File: docs/src/content/docs/hi/index.mdx (hunk 1 above)
Comment (at-the-vr) [inline docs/src/content/docs/hi/index.mdx:49]:
```suggestion
<TestimonialGrid title="जानिए लोग क्या कह रहे हैं">
```

**Review by delucis — APPROVED** [review]
Yay! Thanks for the translation @at-the-vr and the review @HiDeoo 🙌

**Pull request merged**